### PR TITLE
Add documentation for tensorflow model optimization toolkit and update quantization aware training docs.

### PR DESCRIPTION
**Pull request by raghuraman-k**

### CODE REVIEW

File: site/en/tutorials/_toc.yaml [hunk 1]
```diff
@@ -24,16 +24,16 @@ toc:
   - title: Overview
     path: /tutorials/eager/
   - title: Eager execution
-    path: /tutorials/eager/eager_basics.ipynb
+    path: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/notebooks/eager_basics.ipynb
```
Comment (lamberta) [inline site/en/tutorials/_toc.yaml]: Outdated paths.

File: site/en/tutorials/_toc.yaml [hunk 2]
```diff
@@ -115,6 +115,12 @@ toc:
   - title: Large-scale linear models
     path: /tutorials/representation/linear
 
+- title: Model Optimization
```
Comment (lamberta) [inline site/en/tutorials/_toc.yaml]: Please remove this file and the tutorial from this changelist.

**Comment by lamberta** [comment]
Created pull request here for initial organization fixes: https://github.com/raghuraman-k/docs/pull/2
Removes tutorial from this CL.

**Comment by raghuraman-k** [comment]
Hi Mark,
 Thanks for your comments. Ideally, I would like to have the model
optimization page stay at the top level (outside of TFLite) and link to it
as we aim to support use cases beyond mobile with this tool.

Lawrence,
 I will not have the bandwidth to address all these comments in a short
time-frame. Can you help or point me to someone in tech writing who can
help move this forward?

Thanks,
Raghu

**Comment by raghuraman-k** [comment]
Hi Raghu,

I will help with whatever I can. Billy and Mark are the two best contacts
for technical writing.

I spoke to Billy in person and Mark offline. It seems like it's ambitious
to publish three docs in one day, but they're doing the best they can to
review the code before it goes live.

Thanks,

Lawrence Chan
TensorFlow Lite

**Comment by lamberta** [comment]
Hi Raghu,
Please start by merging this pull request: https://github.com/raghuraman-k/docs/pull/2

**Comment by raghuraman-k** [comment]
Sure, will do that, where would the tutorial go ? I see that it is deleted
in this PR.

Thanks,
Raghu

**Comment by raghuraman-k** [comment]
Thanks to everyone. If we can do the following split, hopefully we can all
work in parallel:
- Convert post quantization training into a ipynb: Can anyone help with
this?
- Creating a top level model optimization doc: I can do this
- Move from HTML to md: I was unable to create tables that looked right
with md. Can someone help with this?
- Address other comments : I can do this

Thanks,
raghu

**Comment by MarkDaoust** [comment]
> Convert post quantization training into a ipynb: Can anyone help with
this?

I'll take care of this.

**Comment by lamberta** [comment]
fyi, these docs are staged here: go/tfo-stage/performance/

File: site/en/performance/model_optimization.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
+# Model optimization
+As more machine learning models are deployed to mobile devices,
+inference efficiency has become a critical issue. Where the computational demand
+for *training* grows with the amount of models trained on different
+architectures, the computational demand for *inference* grows in proportion to
+the amount of users. We introduce the <b> Tensorflow Model Optimization Toolkit </b> to minimize complexity of inference: model size, latency and power consumption.
```
Comment (lamberta) [inline site/en/performance/model_optimization.md]: Tensorflow Model Optimization Toolkit ? Should this link somewhere? Is it a separate product or a feature built in to toco or something?

File: site/en/performance/quantization_training.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+<li> Operator fusion at inference time needs to be accurately modeled at training time </li>
+<li> Quantization effects at inference are modeled at training time </li>
+</ol>
+
+
+## Quantization aware training with TensorFlow
+For efficient inference, we combine batch normalization with preceding convolutional/fully connected layers prior to quantization by [folding batch norm layers](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/quantize/python/fold_batch_norms.py). 
+
+We model quantization error by inserting [fake quantization](../api_guides/python/array_ops.md#Fake_quantization) nodes simulate the
+effect of quantization in the forward and backward passes. The forward pass models quantization, while the backward pass models quantization as a straight through estimator (See [Bengio et al.,2013](https://arxiv.org/abs/1308.3432)). Both the forward and backward pass simulate the quantization of weights and activations. (See [Benoit et al.,2017](https://arxiv.org/abs/1712.05877), [Krishnamoorthi,2018](https://arxiv.org/abs/1806.08342))
```
Comment (lamberta) [inline site/en/performance/quantization_training.md]: In general, we're writing engineering docs and not academic docs. Do these papers help users with this? Can you summarize why they would care?

File: site/en/performance/_toc.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- title: Performance
+  path: /performance/
+- title: Performance guide
+  path: /performance/performance_guide
+- title: Input pipeline performance guide
+  path: /performance/datasets_performance
+- title: Benchmarks
+  path: /performance/benchmarks
+
+- heading: Model optimization Toolkit
```
Comment (lamberta) [inline site/en/performance/_toc.yaml]: I'm still unclear what the "Model optimization toolkit" is? Is it a product or techniques? Should this be "Model optimization methods"? (or something) Is there something else to download, install, import, etc?
